plugins/polio/js/src/domains/LQAS-IM/LQAS/LqasAfroOverview/ListView/HasLocationIcon.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FunctionComponent } from 'react';
+import { defineMessages } from 'react-intl';
+import { Tooltip, Box } from '@material-ui/core';
+import LocationDisabledIcon from '@material-ui/icons/LocationDisabled';
+
+import { useSafeIntl } from 'bluesquare-components';
+import ShapeSvg from '../../../../../../../../../hat/assets/js/apps/Iaso/components/svg/ShapeSvgComponent';
+
+const MESSAGES = defineMessages({
+    withShape: {
+        defaultMessage: 'With territory',
+        id: 'iaso.registry.withShape',
+    },
+    withLocation: {
+        defaultMessage: 'With point',
+        id: 'iaso.registry.withLocation',
+    },
+    noGeographicalData: {
+        defaultMessage: 'Without geography',
+        id: 'iaso.label.noGeographicalData',
+    },
+});
+type Props = { shape: any };
```

Comment:
Shape type is here: hat/assets/js/apps/Iaso/domains/orgUnits/types/shapes.ts

Reply:
Not the right type, I'm afraid, but there was already a type in polio constants 😀